pages/_en-US/twilightmenu/faq.md

```diff
@@ -94,3 +94,15 @@ For PNG banners you simply need any PNG file with 15 colors or fewer and a maxim
 banner.bin type banners are able to be animated and allow you to replace the title that will be shown in TWiLight Menu++. They can be made using [NDS Banner Editor](https://github.com/TheGameratorT/NDS_Banner_Editor/releases), it's currently recommended to use the [nightly version](https://nightly.link/TheGameratorT/NDS_Banner_Editor/workflows/build/master) as it has several improvements over the stable release.
 
 Pre-made banners can be found in the [icons section of the TWiLight Menu++ skins site](https://skins.ds-homebrew.com/icon/) and if you make any they can be shared there as well.
+
+#### How do I install custom themes for TWiLight Menu++?
+Custom skins for themes can be acquired from [the official skins site](https://skins.ds-homebrew.com/), whic houses serveral community-made skins ready to apply. You can also make a custom skin for the Nintendo 3DS and Nintendo DSi themes on your own by following [this guide](https://wiki.ds-homebrew.com/twilightmenu/custom-dsi-3ds-skins). The **Homebrew Launcher**, **Sega Saturn** and **Game Boy Color** themes are _**not**_ customizable.
+
+Once you have aquired a custom skin, you can install it by placing its folder on the appropiate path, which depends on what theme the skin is made for.
+- Skins for the Nintendo DSi theme go in `sd:\_nds\TWiLightMenu\dsimenu\themes\`
+- Skins for the Nintendo 3DS theme go in `sd:\_nds\TWiLightMenu\3dsmenu\themes\`
+- Skins for the R4 Original theme go in `sd:\_nds\TWiLightMenu\r4menu\themes\`
+
+If you are using a console from the 3DS family, you can install custom skins using [Universal-Updater](https://github.com/Universal-Team/Universal-Updater/releases). Go to Universal-Updater settings, then `Select Unistore`, `+`, `TWiLight Menu++ Skins`
```

Comment:
I feel like `3DS family console` is better wording than `console from the 3DS family`, it doesn't roll off the tongue right for me.

Reply:
Also, add a period to the end of this line.